wrappers/kratos_CFD_wrapper.py

```diff
@@ -400,18 +391,42 @@ def __importKratosConditions(self, src, dst, entitylist=None):
 
                 self.free_id += 1
 
-            condition_id = self.uuid_to_id_condition_map[condition.uid]
+                condition_id = self.uuid_to_id_condition_map[condition.uid]
+
+                dst.CreateNewCondition(
+                    "WallCondition3D",
+                    condition_id,
+                    [self.uuid_to_id_node_map[p] for p in condition.points],
+                    self.element_properties)
+
+    def importKratosDof(self, src, dst):
+        """ Fix the Dof for the appropiate nodes
 
-            dst.CreateNewCondition(
-                "WallCondition3D",
-                condition_id,
-                [self.uuid_to_id_node_map[p] for p in condition.points],
-                self.element_properties)
+        Iterates over all nodes in the kratos mesh and fix its DoF
+        according with the information of the SimPhoNy mesh.
+
+        """
+
+        # for point in src.iter_points():
+
+        #     node_id = self.uuid_to_id_node_map[point.uid]
+
+        #     node = dst.GetNodes()[node_id]
+
+        #     if point.data[CUBAExtension.IMPOSED_VEL] == 1:
+        #         node.Fix(VELOCITY_X)
+        #         node.Fix(VELOCITY_Y)
+        #         node.Fix(VELOCITY_Z)
+
+        #     if point.data[CUBAExtension.IMPOSED_PRES] == 1:
+        #         node.Fix(PRESSURE)
+
+        pass
 
     # FileIO
 
     def read_modelpart(self, filename):
-        """ Reads a Kratos formated modelpart NYI
+        """ Reads a Kratos formated modelpart
```

Comment:
I would have expected that this method reads in a KRATOS file and returns a SimPhoNy mesh (with name "Model") .  is that its intent?

just wondering about lines 435-437 below:
```
        self.fluid_model_part = ModelPart("FluidPart")

        self.addNodalVariablesToModelpart(self.fluid_model_part)
```